apps/backoffice/src/app/api/utils/io-services-cms-proxy.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  API_SERVICES_CMS_BASE_PATH,
+  API_SERVICES_CMS_MOCKING,
+  API_SERVICES_CMS_URL
+} from "@/config/constants";
+import { Client, createClient } from "@/generated/services-cms/client";
+import { readableReport } from "@pagopa/ts-commons/lib/reporters";
+import * as E from "fp-ts/lib/Either";
+import * as t from "io-ts";
+import { NextRequest, NextResponse } from "next/server";
+
+if (API_SERVICES_CMS_MOCKING) {
+  const { setupMocks } = require("../../../../mocks");
+  setupMocks();
+}
+
+const buildClient = (): Client =>
+  createClient({
+    baseUrl: API_SERVICES_CMS_URL,
+    fetchApi: (fetch as any) as typeof fetch,
+    basePath: API_SERVICES_CMS_BASE_PATH
+  });
+
+type PathParameters = {
+  serviceId?: string;
+  keyType?: string;
+  limit?: string;
+  offset?: string;
+};
+
+export const forwardIoServicesCmsRequest = async <T extends keyof Client>(
```

Comment:
may you add a comment on the purpose of this method?

Reply:
done